lua/nvim-treesitter-playground.lua

```diff
@@ -10,7 +11,15 @@ function M.init()
       module_path = "nvim-treesitter-playground.internal",
       updatetime = 25,
       persist_queries = false
-    }
+    },
+    query_linter = {
+      module_path = "nvim-treesitter-playground.query_linter",
+      use_virtual_text = true,
+      lint_events = {"BufWrite", "CursorHold"},
+      is_supported = function(lang)
+        return lang == 'query' and parsers.has_parser('query')
```

Comment:
Just curious, is there a reason for the mixed single quotes and double quotes? I tend to do this a lot when I'm writing lua as well and I'm not sure if it's a lua convention or not.

Reply:
I prefer single quotes, my auto-formatter double quotes. What should I use?

Reply:
You can use whatever you want. I don't care. I started using double quotes for most things because I go back and forward between a lot of languages and double is standard for most.

Reply:
Should be fixed now.